styles.css

```diff
@@ -13,8 +13,8 @@ nav {
   position: absolute;
   width: 100%;
   height: 72px;
-  left: 0px;
-  top: 0px;
+  left: 0;
```

Comment:
- [ ] Hello @Obimbo07 , Great effort working on your portfolio project (mobile first). But i currently notice the project is not aligning properly to the mobile screen size of `375px` as requested in the project requirement. I would kindly suggest you work on the mobile view of your project as much as possible following the Figma design template given strictly.
Below is the screen recording of how your project looks like in mobile view 👇 

[screen-capture.webm](https://user-images.githubusercontent.com/78634841/235486820-f07beb2c-1dd3-4f09-8a6a-2bceca92118e.webm)